guides/v2.3/index.html

```diff
@@ -4,7 +4,11 @@
 
 <section class="home-features">
   <div class="container">
-
+    
+      <br/>
+      <div class="message-banner index">
+          Looking for access to Magento 2.3 Beta code? See <a href="{{ site.baseurl }}/guides/v2.3/release-notes/2.3.0-quick-start.html">Magento 2.3.0 Beta Quick Start Guide</a> for more information!
```

Comment:
Technically, the link should be `{{ page.baseurl  }}/release-notes/2.3.0-quick-start.html`, but if it works, OK.